recipes/globus-sdk/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "globus-sdk" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/globus/globus-sdk-python/archive/{{ version }}.tar.gz
+  sha256: 202dadb9e011d3846d71638e79717c192bc431ce8e7fc8b900adbfd4e4dd9407
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Could this be a `noarch: python` package instead of skipping Windows?

Reply:
I will try that again.  The Windows build failed but perhaps that's not critical, depending on the reason for it.

Reply:
If it was the Azure win-64 build then that one is an issue with the CI and not the recipe. Provided Appveyor passes it's fine.

Reply:
Okay, that's very good to know.  The Azure builds are pretty new (at least to me) so I wasn't quite sure what to make of the failure.  I've also never tried building this package for Windows myself so had no idea if it was expected to work or not.